.github/ISSUE_TEMPLATE/story.md

```diff
@@ -8,7 +8,7 @@ priority: ''
 ---
 
 ### Describe the story, please be clear on scope of the story.
-Please be conscice with the scope for the story.  
+Please be concice with the scope for the story.  
```

Comment:
```suggestion
Please be concise with the scope for the story.  
```

Reply:
https://www.merriam-webster.com/dictionary/concise

Reply:
Yeah, I noticed that after my initial commit. I see your brain has an auxiliary spell check module too! 😆